src/contract_spec.ts

```diff
@@ -1,13 +1,13 @@
-import { JSONSchema7, JSONSchema7Definition } from "json-schema";
+import type { JSONSchema7, JSONSchema7Definition } from "json-schema";
 import {
   ScIntType,
   XdrLargeInt,
   xdr,
   Address,
   Contract,
   scValToBigInt,
-} from ".";
-import { Ok } from "./rust_types";
+} from "@stellar/stellar-base"
```

Comment:
I think `from "."` correct here

Reply:
we should be very very careful to only reference base in one place, otherwise I'd be worried about the whole `instanceof` thing cropping up again

Reply:
If you import `from "."` here, you get a "Dependency cycle detected (eslint: [import/no-cycle](https://github.com/import-js/eslint-plugin-import/blob/v2.29.1/docs/rules/no-cycle.md))" warning. Cyclic dependencies were causing all sorts of pain when I first exported `ContractClient`, though I can change this particular instance back and everything still builds. However, I noticed that other files throughout, [such as `src/soroban/api.ts`](https://github.com/AhaLabs/js-stellar-sdk/blob/2ee6126c72b36a458787df78b60913c579d42b3c/src/soroban/api.ts#L1), are importing from `@stellar/stellar-base`, which is why I started following suit.